BoxContentSDKSampleApp/Makefile

```diff
@@ -0,0 +1,30 @@
+CARTHAGE := $(shell command -v carthage 2> /dev/null)
+
+.PHONY: help
```

Comment:
All of these rules are technically .PHONY. Maybe we should mark that, too, to be clean.

Reply:
done